dex/src/main/scala/com/wavesplatform/dex/error/MatcherError.scala

```diff
@@ -588,6 +588,9 @@ case class SubscriptionsLimitReached(limit: Int, id: String)
       e"The limit of ${Symbol("limit") -> limit} subscriptions of this type was reached. The subscription of ${Symbol("id") -> id} was stopped"
     )
 
+case class InvalidPublicKey(reason: String)
```

Comment:
We have `UserPublicKeyIsNotValid` which does same thing. But it doesn't contain the `reason`. Suggest:
1. Remove `InvalidPublicKey`
2. Add the `reason` to `UserPublicKeyIsNotValid`
3. Use it :)

Reply:
fixed